packages/web/src/components/gcds-select/gcds-select.stories.tsx

```diff
@@ -56,18 +56,18 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-select
   select-id="${args.selectId}"
   label="${args.label}"
```

Comment:
Same comment about the label: would you mind marking it as required in the story and setting a default data type?